### PR TITLE
WIP: Adding Ambassador Landing Page

### PR DESCRIPTION
**Pull request by ryan-shea**
##### Description

Adding a landing page for the community ambassador program!

##### Checklist

- [ ] linter status: 100% pass
- [ ] changes don't break existing behavior
- [ ] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

Web

##### Testing

It only adds an HTML file that's based off of a preexisting template :)

##### Refers/Fixes

### CODE REVIEW

File: app/retail/templates/ambassadors.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    <div class="container-fluid header dash">
+      {% include 'shared/nav.html' %}
+    </div>
+    <div class="container">
+      <div class="row">
+        <img class="animated fadeIn src="{% static "v2/images/ambassadors.png"%}">
+      </div>
+      <div class="row">
+        <div class="col">
+          <h1>Become a Gitcoin Ambassador</h1>
```
Comment (mbeacom) [inline app/retail/templates/ambassadors.html]: Can you wrap the copy throughout this file in `{% trans "copy" %}` - for example here:  `{% trans "Become a Gitcoin Ambassador" %}`

File: app/retail/templates/ambassadors.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            <ul>
+              <li>Passion for blockchain and open source</li>
+              <li>Knowledge of the Gitcoin Platform</li>
+              <li>Commitment to helping others</li>
+              <li>Esteemed English / written communication ability</li>
+              <li>2-5 hours a week</li>
+            </ul>
+          </div>
+          <div class="col">
+            <a href="https://rshea.typeform.com/to/omAKYw"><button type="button">Apply</button></a>
```
Comment (mbeacom) [inline app/retail/templates/ambassadors.html]: Can you add `target="_blank" rel="noopener noreferrer"` on `a` as well? here as well?

**Comment by thelostone-mc** [comment]
@ryan-shea wanna update this PR addressing the comments and post a screenshot ? ^_^

**Comment by ryan-shea** [comment]
@thelostone-mc yup! will take care of this today

**Comment by thelostone-mc** [comment]
Closing this out , would need a mock design -> accordingly code it up ^_^

**Pull request closed without merging**